### DISCUSSION TITLE
Delete my github pages repo and start from scratch

### DISCUSSION THREAD

**Discussion by tms57** [community/community#65397] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

After a series of catastrophes I've concluded that there is NO POSSIBLE WAY to re-sync my local repo's git file with the remote one, I would just like to completely wipe my **username.github.io** repo and create a new one. (I don't care about lost history - I just want to start over with a new GitHub pages site)
My question before doing this is: 
Once I delete my **username.github.io** repo on GitHub, **_will I be able to create a new one with the same name_**?
If not, what would be my best method to accomplish this?

Thanks.

**Comment by queenofcorgis** [comment]
Hey @tms57 !

Interesting question... you'll be able to delete and recreate the username repository I confirmed with the Pages team 😄
